kitsune/search/search.py

```diff
@@ -368,7 +376,12 @@ def get_highlight_fields_options(self):
 
     def get_filter(self):
         # `should` with `minimum_should_match=1` acts like an OR filter
-        return DSLQ("bool", should=self._from_children("get_filter"), minimum_should_match=1)
+        # Pass the flag down to children filters
+        return DSLQ(
+            "bool",
+            should=[child.get_filter() for child in self._children],
+            minimum_should_match=1,
+        )
```

Comment:
These new lines should be removed, and the original line restored. The `self._from_children("get_filter")` is necessary because it flattens the list of filters.